Dynatrace/dynatrace-operator
# Dynatrace containers livenessprobes switched from `/healthz` to `/livez`

# Description
The `/healthz` endpoint for the livenessprobe is [deprecated since Kubernetes 1.16](https://kubernetes.io/docs/reference/using-api/health-checks/#api-endpoints-for-health). It's replaced by the more specific `/livez`.

## How can this be tested?
Apply dynakube with cloudnativeFullstack:
```yaml
apiVersion: dynatrace.com/v1beta1
kind: DynaKube
metadata:
  name: dk
  namespace: dynatrace
spec:
  apiUrl: <api_url>
  activeGate:
    capabilities:
    - kubernetes-monitoring
    - routing
  oneAgent:
    cloudNativeFullStack: {}
```
- see if all pods in `dynatrace` namespace are working correctly.
- check is no probe problems are visible in `kubectl get events --sort-by='.lastTimestamp' -n dynatrace` output
- check if `/healthz` is no longer used: `kubectl -n dynatrace get pod -o yaml | grep healtz | wc -l`

## Checklist
